Magento_Customer/templates/form/edit.phtml

```diff
@@ -56,16 +56,16 @@
         </div>
 
         <div class="checkbox dashboard-form__divider">
-            <input type="checkbox" 
-                   name="change_password" 
-                   id="change-password" 
-                   data-role="change-password" 
-                   value="1" 
+            <input type="checkbox"
+                   name="change_password"
+                   id="change-password"
+                   data-role="change-password"
+                   value="changePassword"
```

Comment:
why we are using changePassword instead of (0,1)

Reply:
same answer as higher